src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,30 +43,34 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${todoId}`}
             </div>
 
-            {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
             <button
               type="button"
               className="delete"
               data-cy="modal-close"
+              aria-label="close"
+              onClick={() => {
+                selectedTodo(0);
+                setLoading(false);
```

Comment:
don't need set loading to false here. Your modal doesn’t render if you reset selected todo id